site/installation.md

```diff
@@ -64,64 +64,64 @@ Verify that you have the right image by running `docker images | grep webloogic-
+| `remoteDebugNodePortEnabled` | Controls whether or not the operator will start a Java remote debug server on the provided port and suspend execution until a remote debugger has attached. | `false` |
+| `serviceAccount`| The name of the service account that the operator will use to make requests to the Kubernetes API server. | `weblogic-operator` |
+| `targetNamespaces` | A list of the Kubernetes namespaces that may contain WebLogic domains that the operator will manage.  The operator will not take any action against a domain that is in a namespace not listed here. | `default` |
+| `weblogicOperatorImage` | The Docker image containing the operator code. | `container-registry.oracle.com/middleware/weblogic-kubernetes-operator:latest` |
+| `weblogicOperatorImagePullPolicy` | The image pull policy for the operator Docker image.  Allowed values are: `Always`, `Never` and `IfNotPresent` | `IfNotPresent` |
+| `weblogicOperatorImagePullSecretName` | Name of the Kubernetes secret to access the Docker Store to pull the WebLogic Server Docker image.  The presence of the secret will be validated when this parameter is enabled. | |
 
 Review the default values to see if any need to be updated to reflect the target environment.  If so, then make a copy of the input file and modify it.  Otherwise, you can use the default input file.
 
 ## Decide which REST configuration to use
 
 The operator provides three REST certificate options:
 
-*	`NONE` will disable the REST server.
-*	`SELF_SIGNED_CERT` will generate self-signed certificates.
-*	`CUSTOM_CERT` provides a mechanism to provide certificates that were created and signed by some other means.
+*	`NONE` Disables the REST server.
+*	`SELF_SIGNED_CERT` Generates self-signed certificates.
+*	`CUSTOM_CERT` Provides a mechanism to provide certificates that were created and signed by some other means.
 
 ## Decide which options to enable
 
 The operator provides some optional features that can be enabled in the configuration file.
 
-### Load Balancing
+### Load balancing with an Ingress controller or a web server
 
-The operator can install the Traefik Ingress provider to provide load balancing for web applications running in WebLogic clusters.  If enabled, an instance of Traefik and an Ingress will be created for each WebLogic cluster.  Additional configuration is performed when creating the domain.
+You can choose a load balancer provider for your WebLogic domains running in a Kubernetes cluster. Please refer to Load balancing with Voyager Ingress Controller, [Load balancing with Traefik Ingress Controller](site/traefik.md), and [Load balancing with Apache Web Server](site/apache.md) for information about the current capabilities and setup instructions for each of the supported load balancers.
 
-Note that the technology preview release provides only basic load balancing:
+Note that this release provides only basic load balancing:
 
 *	Only HTTP(S) is supported. Other protocols are not supported.
```

Comment:
I think we need to identify which other protocols are not supported, T3, IIOP? I think now that we support HAPROXY and Apache we can remove the line
"Note that this release provides only basic load balancing:"
We need Shean to check this statement.

Reply:
Ok, I'll remove this line, ""Note that this release provides only basic load balancing:"

How do we get "Shean to check this statement."

Reply:
I tried adding him for review but was not able.  I will have a meeting and discuss these 2 statements and then confirm with you if we should remove.